sample/golang-echo/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.9-alpine
```

Comment:
Docker have official Golang images, are you certain that the LTS Golang version is vulnerable? Can you link to something showing that?